lib/_inspect.js

```diff
@@ -139,6 +92,56 @@ function portIsFree(host, port, timeout = 2000) {
   });
 }
 
+function runScript(script, scriptArgs, inspectHost, inspectPort, childPrint) {
+  return portIsFree(inspectHost, inspectPort)
```

Comment:
Moving the port check here so it's skipped for remote debugging. Waiting or the the port to be available doesn't make sense for `node inspect <host>:<port>`/`node inspect -p <pid>`.